config/schema/document_types/policy.json

```diff
@@ -1,5 +1,7 @@
 {
   "fields": [
-    "slug"
+    "slug",
+    "organisations",
```

Comment:
This isn't needed, since "organisations" is already in the `base_document_type.json` schema (on directory level up), which is merged with this one.

Adding "people" is needed, though, since that one isn't in the base.